<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>happyTutor</class>
 <widget class="QMainWindow" name="happyTutor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TutorMode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:blue;
border: 1px solid black;
color:white;
}
</string>
    </property>
    <property name="text">
     <string>back to tutor menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,215,0);
}
QPushButton:checked{
background: rgb(255,215,0);
}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="aButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,182,193);
}

QPushButton:checked{
background: rgb(255,182,193);
}
    
if (aButton-&gt;isOn())
       btnAbort-&gt;setPaletteBackgroundColor( QColor(255,182,193) );
else
       btnAbort-&gt;setPaletteBackgroundColor(
    this-&gt;paletteBackgroundColor() );
</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="eButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,0,0);
}
QPushButton:checked{
background: rgb(255,0,0);
}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="hButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(113,198,113);
}
QPushButton:checked{
background: rgb(113,198,113);
}</string>
    </property>
    <property name="text">
     <string>H</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="apostButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(75,0,130);
}
QPushButton:checked{
background: rgb(75,0,130);
}</string>
    </property>
    <property name="text">
     <string>'</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(205,16,118);
}
QPushButton:checked{
background: rgb(205,16,118);
}</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="semiButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,0,255);
}
QPushButton:checked{
background: rgb(255,0,255);
}</string>
    </property>
    <property name="text">
     <string>;</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>If You're Happy and You Know It</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(122,197,205);
}
QPushButton:checked{
background: rgb(122,197,205);
}</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,193,37);
}
QPushButton:checked{
background: rgb(255,193,37);
}</string>
    </property>
    <property name="text">
     <string>T</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="uButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(0,139,69);
}
QPushButton:checked{
background: rgb(0,139,69);
}</string>
    </property>
    <property name="text">
     <string>U</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(16,78,139);
}
QPushButton:checked{
background: rgb(16,78,139);
}</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,255,0);
}
QPushButton:checked{
background: rgb(255,255,0);
}</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,130,171);
}
QPushButton:checked{
background: rgb(255,130,171);
}</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,128,0);
}
QPushButton:checked{
background: rgb(255,128,0);
}</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="jButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(0,100,0);
}
QPushButton:checked{
background: rgb(0,100,0);
}</string>
    </property>
    <property name="text">
     <string>J</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(30,144,255);
}
QPushButton:checked{
background: rgb(30,144,255);
}</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(205,55,0);
}
QPushButton:checked{
background: rgb(205,55,0);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(0,255,255);
}
QPushButton:checked{
background: rgb(0,255,255);
}</string>
    </property>
    <property name="text">
     <string>K</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="playSong">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:blue;
border: 1px solid black;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Listen to Song</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopSong">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:blue;
border: 1px solid black;
color:white;
}</string>
    </property>
    <property name="text">
     <string>Stop Song</string>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>aButton</zorder>
   <zorder>hButton</zorder>
   <zorder>apostButton</zorder>
   <zorder>sButton</zorder>
   <zorder>semiButton</zorder>
   <zorder>label</zorder>
   <zorder>gButton</zorder>
   <zorder>fButton</zorder>
   <zorder>jButton</zorder>
   <zorder>lButton</zorder>
   <zorder>dButton</zorder>
   <zorder>kButton</zorder>
   <zorder>playSong</zorder>
   <zorder>stopSong</zorder>
   <zorder>uButton</zorder>
   <zorder>pButton</zorder>
   <zorder>tButton</zorder>
   <zorder>wButton</zorder>
   <zorder>yButton</zorder>
   <zorder>eButton</zorder>
   <zorder>oButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>happyTutor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSong</sender>
   <signal>clicked()</signal>
   <receiver>happyTutor</receiver>
   <slot>playSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopSong</sender>
   <signal>clicked()</signal>
   <receiver>happyTutor</receiver>
   <slot>stopSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>594</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>playSong()</slot>
  <slot>stopSong()</slot>
 </slots>
</ui>
